### ISSUE TITLE
INLINE_GROUPED_CLASSES=YES fails with GROUP_NESTED_COMPOUNDS=NO

### ISSUE THREAD

**Issue by yuriishutkin** [doxygen/doxygen#10350]
**Describe the bug**
If define INLINE_GROUPED_CLASSES=YES and there are nested classes inside a group, inconsistent refman.tex is generated - inner class apears in class documentation as a separate input entry but it is not generated. As a result latex fails.

**Expected behavior**
Nested classes inside a group with INLINE_GROUPED_CLASSES=YES do not appear as separated input entry in class documentation.

**To Reproduce**
Set INLINE_GROUPED_CLASSES=YES (keep GROUP_NESTED_COMPOUNDS=NO) and run the following file:

```
/**
 *  \defgroup test_group Test group
 *  \{
 */
 
//! TestOuter
class TestOuter {
public:
    //! TestInner
    class TestInner {
	public:
        TestInner()  {
        }
    };
};
 
/**
 *  \}
 */
```

in refman.tex:
```
\chapter{Class Documentation}
\input{class_test_outer_1_1_test_inner}
```

Manual setting `\ingroup test_group` for the inner class or setting GROUP_NESTED_COMPOUNDS=YES globally solves the issue but puts all nested compounds to the group class list that can be not intended.

**Version**
1.9.8

**Comment by albert-github** [comment]
I also see problems with the other output formats.
Using as Doxyfile just:
```
QUIET = YES
INLINE_GROUPED_CLASSES=YES
GROUP_NESTED_COMPOUNDS=NO
```

**Comment by albert-github** [comment]
Looks like the problem is introduced by:
```
Commit: 3b314b5d856f6f7a25d512737e640a6b7b17c8d4 [3b314b5]
Date: Monday, July 27, 2015 6:29:52 PM
Actually using value of GROUP_NESTED_COMPOUNDS option
```
though before the `GROUP_NESTED_COMPOUNDS` was not taken into account at all so not correct either.

Might be related to #5445 and #5814

@doxygen any idea?

**Comment by doxygen** [comment]
@yuriishutkin Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official version is released.

**Comment by yuriishutkin** [comment]
Confirm. It works fine now.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.10.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
